build-logic-commons/gradle-plugin/src/main/kotlin/gradlebuild.code-quality.gradle.kts

```diff
@@ -24,6 +14,26 @@ import org.gradle.util.internal.VersionNumber
  * limitations under the License.
  */
 
+import gradle.kotlin.dsl.accessors._16244812a8df0afe05037003e4f0274c.check
```

Comment:
this one finally making the file compile always having this file with error, this ide suggestion.